examples/cloudflare-workers/src/index.ts

```diff
@@ -1,8 +1,117 @@
-import type { Env } from "hono/types";
-import app from "./router";
-
-export default {
-  async fetch(request: Request, environment: Env, context: ExecutionContext): Promise<Response> {
-    return app.fetch(request, environment, context);
-  },
-};
+import { Hono } from "hono";
+import { Index } from "@upstash/vector";
+import { openai, RAGChat, upstash } from "@upstash/rag-chat";
+
+const app = new Hono<{
+  Variables: {
+    ragChat: RAGChat;
+  };
+  Bindings: {
+    UPSTASH_REDIS_REST_URL: string;
+    UPSTASH_REDIS_REST_TOKEN: string;
+    UPSTASH_VECTOR_REST_URL: string;
+    UPSTASH_VECTOR_REST_TOKEN: string;
+    QSTASH_TOKEN: string;
+    OPENAI_API_KEY: string;
+  };
+}>();
+
+app.use("*", async (c, next) => {
+  const ragChat = new RAGChat({
+    // model: upstash("meta-llama/Meta-Llama-3-8B-Instruct", { apiKey: c.env.QSTASH_TOKEN }),
```

Comment:
we can put the model back in, and comment out the alternative below

Reply:
Oh i missed this thanks